src/ProtectedProxies/ProtectedERC1967Proxy.sol

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: UNLICENSED
+// (c) SphereX 2023 Terms&Conditions
+
+pragma solidity ^0.8.0;
+
+import {ERC1967Proxy, Proxy} from "openzeppelin/Proxy/ERC1967/ERC1967Proxy.sol";
+
+import {SphereXProtectedProxy} from "../SphereXProtectedProxy.sol";
```

Comment:
It's really small, but I don't like when file `a` imports or depends on a file that is above it in the FS hirarchy

Reply:
I think @idoh-spherex tried to mimic OZ structure
https://github.com/OpenZeppelin/openzeppelin-contracts/blob/181d518609a9f006fcb97af63e6952e603cf100e/contracts/proxy/ERC1967/ERC1967Proxy.sol#L6